package.json

```diff
@@ -8,6 +8,7 @@
     "dev-cli": "node -r esm ./scripts/cli.js",
     "lint": "eslint ./src --ext .js,.ts,.tsx,.mdx --max-warnings 0",
     "lint:fix": "npm run lint -- --fix",
+    "int": "testcafe chrome",
```

Comment:
Why is this needed?

Reply:
i cant find on how to run a specific single test in testcafe, so now what i do is `npm run int <path-to-test>`, the path i copy paste relative path via vsc.

Reply:
My concern is "int" doesnt say what you just did. Might want to have a better name for it and maybe add into the readme what it does?